SonarQube.Client.Tests/SonarQubeService_GetHotspotAsync.cs

```diff
@@ -0,0 +1,185 @@
+    ""vulnerabilityProbability"": ""LOW""
+  },
+  ""status"": ""TO_REVIEW"",
+  ""line"": 10,
+  ""message"": ""message"",
+  ""assignee"": ""joe"",
+  ""author"": ""joe"",
+  ""creationDate"": ""2020-01-02T15:43:10+0100"",
+  ""updateDate"": ""2020-01-02T15:43:10+0100"",
+  ""changelog"": [
+    {
+      ""user"": ""joe"",
+      ""userName"": ""Joe"",
+      ""creationDate"": ""2020-01-02T14:44:55+0100"",
+      ""diffs"": [
+        {
+          ""key"": ""diff-key-0"",
+          ""newValue"": ""new-value-0"",
+          ""oldValue"": ""old-value-0""
+        }
+      ],
+      ""avatar"": ""my-avatar"",
+      ""isUserActive"": true
+    },
+    {
+      ""user"": ""joe"",
+      ""userName"": ""Joe"",
+      ""creationDate"": ""2020-01-02T14:44:55+0100"",
+      ""diffs"": [
+        {
+          ""key"": ""diff-key-1"",
+          ""newValue"": ""new-value-1"",
+          ""oldValue"": ""old-value-1""
+        }
+      ],
+      ""avatar"": ""my-avatar"",
+      ""isUserActive"": true
+    },
+    {
+      ""user"": ""joe"",
+      ""userName"": ""Joe"",
+      ""creationDate"": ""2020-01-02T14:44:55+0100"",
+      ""diffs"": [
+        {
+          ""key"": ""diff-key-2"",
+          ""newValue"": ""new-value-2"",
+          ""oldValue"": ""old-value-2""
+        }
+      ],
+      ""avatar"": ""my-avatar"",
+      ""isUserActive"": true
+    }
+  ],
+  ""comment"": [
+    {
+      ""key"": ""comment-0"",
+      ""login"": ""Joe"",
+      ""htmlText"": ""html text 0"",
+      ""markdown"": ""markdown 0"",
+      ""createdAt"": ""2020-01-02T14:47:47+0100""
+    },
+    {
+      ""key"": ""comment-1"",
+      ""login"": ""Joe"",
+      ""htmlText"": ""html text 1"",
+      ""markdown"": ""markdown 1"",
+      ""createdAt"": ""2020-01-02T14:47:47+0100""
+    },
+    {
+      ""key"": ""comment-2"",
+      ""login"": ""Joe"",
+      ""htmlText"": ""html text 2"",
+      ""markdown"": ""markdown 2"",
+      ""createdAt"": ""2020-01-02T14:47:47+0100""
+    }
+  ],
+  ""users"": [
+    {
+      ""login"": ""joe"",
+      ""name"": ""Joe"",
+      ""active"": true
+    }
+  ],
+  ""canChangeStatus"": true
+}");
+
+            var result = await service.GetHotspotAsync(hotspotKey, CancellationToken.None);
+
+            messageHandler.VerifyAll();
+
+            result.HotspotKey.Should().Be(hotspotKey);
+            result.Message.Should().Be("message");
+            result.Assignee.Should().Be("joe");
+            result.Status.Should().Be("TO_REVIEW");
+            result.Line.Should().Be(10);
+
+            result.Organization.Should().Be("default-organization");
+            result.ProjectKey.Should().Be("com.sonarsource:test-project");
+            result.ProjectName.Should().Be("test-project");
+
+            result.ComponentKey.Should().Be("com.sonarsource:test-project:src/main/java/com/sonarsource/FourthClass.java");
+            result.ComponentPath.Should().Be("src/main/java/com/sonarsource/FourthClass.java");
```

Comment:
Perhaps ComponentPath should be FilePath, don't know if it's better to stay loyal to the returned values, or to make it more explicit for us to understand.

Reply:
Since I'm expecting the API to change I've done the absolute minimum necessary now to give us something we can consume in the client. I didn't think it was worth doing any extra work at this point.